modules/server/src/network/resolvers/index.ts

```diff
@@ -35,7 +35,7 @@ export const createResolvers = (configs: NodeConfig[]) => {
 		 * Checks validity of SQON
 		 * For now we will pass through the non SQON object to the pipeline
 		 *
-		 * TODO: resolve Arranger / SQONBuilder SQON outer wrapper conflict
+		 * // TODO: resolve Arranger / SQONBuilder SQON outer wrapper conflict
```

Comment:
curious double comment. no action needed, just wondering.

Reply:
Great q. The TODO for vscode plugin picks up `//` but not how it was originally.